hazelcast/src/main/java/com/hazelcast/internal/management/request/ChangeClusterStateRequest.java

```diff
@@ -0,0 +1,64 @@
+package com.hazelcast.internal.management.request;
+
+import com.eclipsesource.json.JsonObject;
+import com.hazelcast.cluster.ClusterState;
+import com.hazelcast.internal.management.ManagementCenterService;
+
+import java.io.IOException;
+
+import static com.hazelcast.util.JsonUtil.getString;
+
+public class ChangeClusterStateRequest implements AsyncConsoleRequest {
+
+    private String state;
+
+    public ChangeClusterStateRequest() {
+    }
+
+    public ChangeClusterStateRequest(String state) {
+        this.state = state;
+    }
+
+    @Override
+    public int getType() {
+        return ConsoleRequestConstants.REQUEST_TYPE_CHANGE_CLUSTER_STATE;
+    }
+
+    @Override
+    public Object readResponse(JsonObject in) throws IOException {
+        return getString(in, "result", "FAIL");
+    }
+
+    @Override
+    public void writeResponse(ManagementCenterService mcs, JsonObject out) throws Exception {
+        JsonObject result = new JsonObject();
+        String resultString = "SUCCESS";
+        try {
+            if (state.equalsIgnoreCase("ACTIVE")) {
+                mcs.getHazelcastInstance().getCluster().changeClusterState(ClusterState.ACTIVE);
```

Comment:
You can pull out `mcs.getHazelcastInstance().getCluster()` to a local var, that will make code cleaner to eyes.